addons/mail/static/tests/mock_server/mock_models/discuss_channel.js

```diff
@@ -418,10 +418,16 @@ export class DiscussChannel extends models.ServerModel {
             }
             const memberOfCurrentUser = this._find_or_create_member_for_self(channel.id);
             if (memberOfCurrentUser) {
+                const message_unread_counter = this.env[
+                    "discuss.channel.member"
+                ]._compute_message_unread_counter([memberOfCurrentUser.id]);
+                this.env["discuss.channel.member"].write([memberOfCurrentUser.id], {
```

Comment:
Could use `DiscussChannelMember` rather than `this.env["discuss.channel.member"]`, for proper typing